bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/test/java/com/google/cloud/bigtable/hbase/TestBigtableConnection.java

```diff
@@ -33,4 +40,15 @@
   public void testBigtableConnectionExists() {
     Assert.assertEquals(BigtableConnection.class, BigtableConfiguration.getConnectionClass());
   }
+
+  @Test
+  public void testTable() throws IOException {
```

Comment:
This block seems duplicate, can it be moved to the common test module?

Reply:
This is definitely duplicated, but that's ok.  This is a unit test. The common test module is for integration tests only.